app/controllers/air_zones_controller.rb

```diff
@@ -4,14 +4,17 @@
 # Controller class for final steps of checking the vehicle compliance
 #
 class AirZonesController < ApplicationController
+  include VrnSession
```

Comment:
```suggestion
  include VrnSessionManipulator
```
Would give us more info about what going on in concern